src/planscape/collaboration/models.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.gis.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import gettext_lazy as _
+from django.contrib.auth import get_user_model
+import uuid
+
+User = get_user_model()
+
+
+class Role(models.TextChoices):
+    CREATOR = "Creator"
+    OWNER = "Owner"
+    COLLABORATOR = "Collaborator"
+    VIEWER = "Viewer"
+
+
+class Permissions(models.Model):
```

Comment:
It might be nice to add the CreatedAtMixin, UpdatedAtMixin that we have for other models, since they provide us a standard way to store created_at and updated_at fields

Reply:
good idea!